.github/workflows/stale.yml

```diff
@@ -57,14 +59,20 @@ jobs:
 
           # Labels on issues exempted from stale.
           exempt-issue-labels: |
-            "Status: Blocked,Status: Decision Required,Peloton 🚴‍️"
+            "Status: Blocked,Status: Decision Required,Peloton 🚴‍♂️"
 
           # Labels on prs exempted from stale.
           exempt-pr-labels: |
-            "Status: Blocked,Status: Decision Required,Peloton 🚴‍"
+            "Status: Blocked,Status: Decision Required,Peloton 🚴‍♂️"
+
+          # Max number of operations per run.
+          operations-per-run: 30
 
           # Remove stale label from issues/prs on updates/comments.
           remove-stale-when-updated: true
 
+          # Order to get issues/PRs.
+          ascending: true
```

Comment:
Let's go oldest first... seems like the best behaviour to adopt for a higher stale hit ratio.